src/univariate/continuous/maxwellboltzmann.jl

```diff
@@ -0,0 +1,63 @@
+"""
+    MaxwellBoltzmann(a)
+
+The *Maxwell-Boltzmann distribution* has probability density function
+```math
+f(x; a) = \\sqrt{\\frac{2}{\\pi}}\\frac{x^2}{a^3}\\exp\\left(-\\frac{x^2}{2a^2}\\right)
+```
+where the parameter
+```math
+a = \\sqrt{\\frac{kT}{m}}
+```
+
+```julia
+MaxwellBoltamann()        # Maxwell-Boltzmann distribution with a = 1
```

Comment:
```suggestion
MaxwellBoltzmann()        # Maxwell-Boltzmann distribution with a = 1
```
Another typo here, less important this time.